packages/CB-serverless-backend/api/groceries/stock.js

```diff
@@ -0,0 +1,33 @@
+import AWS from 'aws-sdk';
+
+import awsConfigUpdate from '../../utils/awsConfigUpdate';
+import getErrorResponse from '../../utils/getErrorResponse';
+import getSuccessResponse from '../../utils/getSuccessResponse';
+
+awsConfigUpdate();
+
+export const updateStock = async (event, context, callback) => {
+  context.callbackWaitsForEmptyEventLoop = false;
+	const dataToUpdate = JSON.parse(event.body);
+  const documentClient = new AWS.DynamoDB.DocumentClient();	
+	const promiseArray = [];
+
+	dataToUpdate.forEach(rowData => {
+		const params = {
+			TableName : 'grocery',
+			Key: {
+				groceryId: parseInt(rowData.groceryId),
+			},
+			UpdateExpression: `set availableQty=:updatedQty`,
+			ExpressionAttributeValues:{
+				":updatedQty":rowData.availableQty,
```

Comment:
Should we add with the existing stock or just fix the stock as this ?

Reply:
the client should always send the updated value. Like if the current value is 12 and if it wants to add 2 more to stock, then the client should send 14 as value. If not there will be unnecessary read for every update.